pad.js

```diff
@@ -0,0 +1,29 @@
+Array.prototype.pad = function (n = -1, padding = null) {
```

Comment:
this is an interesting way of doing this! obviously, this is completely functional so it works, but you could also just add these individual methods to the `Array` object itself